sos/report/plugins/redis.py

```diff
@@ -35,6 +35,16 @@ def setup(self):
             self.var_puppet_gen + "/etc/security/limits.d/"
         ])
 
+        # rh-redis32 collection from scl
+        self.add_copy_spec([
+            "/etc/opt/rh/rh-redis32/redis.conf",
+            "/etc/opt/rh/rh-redis32/redis.conf.puppet",
+            "/etc/opt/rh/rh-redis32/redis-sentinel.conf",
+            "/etc/opt/rh/rh-redis32/redis-sentinel.conf.puppet",
+            "/var/opt/rh/rh-redis32/log/redis/sentinel.log",
+            "/var/opt/rh/rh-redis32/log/redis/redis.log"
+            ])
```

Comment:
I think this would be better implemented via the `SCLPlugin` interface. The `postgresql` plugin has a good reference for SCLs.